inst/scripts/importPDB.R

```diff
@@ -0,0 +1,158 @@
+# importPDB.R
+# Non-exported function
+
+#=============================STOP & SAVE======================================
+# Provide a mechanism to stop:

```

Comment:
I don't know yet. It's an interesting idea. Leave it in so we can play with it. We can always comment it out.